pkgs/top-level/all-packages.nix

```diff
@@ -18767,7 +18767,7 @@ in
 
   ant-bloody-theme = callPackage ../data/themes/ant-theme/ant-bloody.nix { };
 
-  ant-dracula-theme = callPackage ../data/themes/ant-theme/ant-dracula.nix { };
+  dracula-theme = callPackage ../data/themes/dracula-theme { };
```

Comment:
This will break existing configurations. Maybe there should be an error thrown when using the old package that tells you to use the new one and to change the theme name in your config

Reply:
I believe the alias should add an appropriate warning